django/thunderstore/repository/models/package.py

```diff
@@ -257,6 +262,18 @@ def post_save(sender, instance, created, **kwargs):
     def post_delete(sender, instance, **kwargs):
         invalidate_cache_on_commit_async(CacheBustCondition.any_package_updated)
 
+    @staticmethod
+    def ensure_namespace(sender, instance, **kwargs):
```

Comment:
This method shouldn't probably exist, we should instead make sure whatever creates packages (the package submission form in this case) properly accounts for namespaces.